### PR TITLE
exceptions branches added to UseTest.java

### LINKED ISSUE #207: Add Exception branches to UseTest

### PR DESCRIPTION
**Pull request by SerJimmy**

### CODE REVIEW

File: src/test/java/org/fundacionjala/at15/pokemon/commands/behavior/UseTest.java [hunk 1]
```diff
@@ -31,4 +33,54 @@ public void itShouldUseMovement() throws IncompleteArguments {
...
+            Use params2 = new Use();
+            params2.setMoveAttack(null);
+            params2.call();
+
+            fail();
+            } catch (IncompleteArguments ex) {
+
+            Assert.assertEquals("Error. Incomplete arguments for the Use command. Required arguments: -move.",ex.getMessage());
+        }
+    }
```
Comment (eliosfg) [inline src/test/java/org/fundacionjala/at15/pokemon/commands/behavior/UseTest.java]: Looks good! but I think you should delete the junk files created.
You can delete it using the function like this: 
`FileEraser.eraseFile(params.getIdPokemonCreated());`
You have 2 pokemos created by "params" and "params1"

File: src/test/java/org/fundacionjala/at15/pokemon/commands/behavior/UseTest.java [hunk 2]
```diff
@@ -31,4 +33,54 @@ public void itShouldUseMovement() throws IncompleteArguments {
...
+            params2.call();
+
+
+            fail();
+        } catch (IncompleteArguments ex) {
+
+            Assert.assertEquals("Your current Pokemon does not know the move: tackle\n" +
+                    "Please, use only learned moves known by your Pokemon.",ex.getMessage());
+        }
+    }
```
Comment (eliosfg) [inline src/test/java/org/fundacionjala/at15/pokemon/commands/behavior/UseTest.java]: Also here, please delete the junk files.

**Review by eliosfg — CHANGES_REQUESTED** [review]

File: src/test/java/org/fundacionjala/at15/pokemon/commands/behavior/UseTest.java (hunk 1 above)
Comment (SerJimmy) [reply src/test/java/org/fundacionjala/at15/pokemon/commands/behavior/UseTest.java]: Done

File: src/test/java/org/fundacionjala/at15/pokemon/commands/behavior/UseTest.java (hunk 2 above)
Comment (SerJimmy) [reply src/test/java/org/fundacionjala/at15/pokemon/commands/behavior/UseTest.java]: Done

**Review by eliosfg — APPROVED** [review]

**Pull request merged**